pkg/operator/ceph/object/admin.go

```diff
@@ -410,6 +408,22 @@ func periodWillChange(current, staged string) (bool, error) {
 	return (diff != ""), nil
 }
 
+// convert a cmp.Path to a path in json format like might be used with jq to query the item
+// e.g., {"a": {"b": "c"}} returns .a.b
+func toJsonPath(path cmp.Path) string {
```

Comment:
Unit test in progress?

Reply:
It's tested functionally by TestCommitConfigChanges()